tests/dialects/test_riscv.py

```diff
@@ -147,12 +147,12 @@ def test_immediate_i_inst():
     # I-Type - 12-bits immediate
     a1 = TestSSAValue(riscv.RegisterType(riscv.Registers.A1))
 
-    pos_invalid_addi_op = riscv.AddiOp(a1, 1 << 11, rd=riscv.Registers.A0)
     with pytest.raises(VerifyException):
+        pos_invalid_addi_op = riscv.AddiOp(a1, 1 << 11, rd=riscv.Registers.A0)
```

Comment:
Hmm, either it should raise on verify or on init, having both calls in the `with` block is confusing